conn.go

```diff
@@ -217,15 +217,18 @@ func (c *Conn) handleMail(arg string) {
 		c.SetUser(user)
 	}
 
+	mailFromRE := "(?i)^FROM:\\s*<((?:\\\\>|[^>])+|\"[^\"]+\"@[^>]+)>  ( [\\w= ]+)?$"
+	if c.server.LenientSMTP {
+		mailFromRE = "(?i)^FROM:[ <]*([^ >]*)[ >]*( [\\w=   ]+)?$"
+	}
 	// Match FROM, while accepting '>' as quoted pair and in double quoted strings
 	// (?i) makes the regex case insensitive, (?:) is non-grouping sub-match
-	re := regexp.MustCompile("(?i)^FROM:\\s*<((?:\\\\>|[^>])+|\"[^\"]+\"@[^>]+)>( [\\w= ]+)?$")
+	re := regexp.MustCompile(mailFromRE)
```

Comment:
Hmmm, this regexp is a leftover from the past I really want to get rid of. Possible to parse this with `strings` instead?

Reply:
I honestly don't know a huge amount about the ESMTP extensions to MAIL FROM but I can go look.

Reply:
Hmm, I'll just create a follow-up issue for this one if you don't feel like fixing it right now.

Reply:
Done. That removes the need for regexp everywhere.